platforms/gcp-dataproc-classical-airgapped/spark_healthcare_nlp_classical.py

```
@@ -0,0 +1,77 @@
+import json, os
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import

(F401)